packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -16,6 +16,9 @@ import 'debug.dart';
 import 'object.dart';
 
 /// How overflowing text should be handled.
+///
+/// A [TextOverflow] should be passed to a [RenderParagraph] widget via its
```

Comment:
nit: You can also use the TextOverflow in other places. Maybe replace the "should" with "can"?